.gitmodules

```diff
@@ -24,9 +24,6 @@
 [submodule ".github/actions/file-changes-action"]
 	path = .github/actions/file-changes-action
 	url = https://github.com/trilom/file-changes-action
-[submodule ".github/actions/cancel-workflow-action"]
```

Comment:
I'd still recommend to leave submodules here even if my action is allow-listed by INFRA. It has the nice side effect that if you update to newer version GitHub will show the diff for example.

Reply:
But should I refer to your action via local files though? That would require check out the whole repo which I'd prefer to avoid if possible.

Reply:
HJow it is done is great now. The submodule works perfectly well in this case  - it will only checkout the actions in CI and you do not have to worry about submodules locally unless you want to update to newer version

Reply:
I thought `@actions/checkout` will check out the whole repo and submodules recursively? Thought there is certain overhead here that we can potentially avoid if using the action directly.

Reply:
Very small, if any. Checkout will indeed checkout the action, but if you are accessing it directly it also has to be downloaded (since you are using the TAG or SHA commit, I believe it happens by just cloning the repo anyway).

Default 'fetch-depth' of checkout action is "1" which means that it will only checkout the exact single commit from the repository and I believe it will also apply for the submodules (depending how it is done by the action), so effectively checking the submodules repo in this case is the same as just running the action from the URL.